.github/workflows/copy-workflow.yml

```diff
@@ -72,6 +72,14 @@ jobs:
           git rm -r .circleci
           git commit -m "disable CircleCI"
         fi
+    - name: remove GX on initial deployment
+      if: ${{ env.INITIAL_TEST_DEPLOYMENT == 1 }}
+      run: |
+        cd $TARGET_REPO_DIR
+        if [[ -d .gx ]]; then
+          git rm -r .gx
+          git commit -m "disable GX"
```

Comment:
```suggestion
          git commit -m "remove .gx"
```

Reply:
In other places the wording used is `disable` should I update the rest to reflect the feedback here while at it?

Reply:
Travis / CircleCI run on every commit, so I thought it makes sense to "disable" them. gx doesn't run, so I thought that "remove" is more precise.

Reply:
ah that makes sense; please ignore me. thanks 🙏